rust/src/shards/gui/widgets/listbox.rs

```diff
@@ -131,7 +144,33 @@ impl Shard for ListBox {
       }
     }
 
-    Ok(common_type::any)
+    let input_type = data.inputType;
+    let slice = unsafe {
+      let ptr = input_type.details.seqTypes.elements;
+      std::slice::from_raw_parts(ptr, input_type.details.seqTypes.len as usize)
+    };
+
+    let element_type = match slice.len() {
+      0 => common_type::none,
+      1 => slice[0],
+      _ => {
+        if slice.iter().skip(1).all(|t| *t == slice[0]) {
```

Comment:
As said on other comment while this is correct it would not be correct to have multiple seqTypes of same type. So in theory this should not be needed. We should verify correctness at some point (until I worked alone I tried to apply this). But anyway out of this PR scope.